### ISSUE TITLE
Que el dump de imagenes se baje con el dump de articulos

### ISSUE THREAD

**Issue by hugoruscitti** [PyAr/CDPedia#128]
_From [facundob...@gmail.com](https://code.google.com/u/113918106348664864973/) on July 23, 2011 10:34:02_

No tener mas el download en la generación, sino que se bajen todas una vez, cuando se bajan los artículos.

_Original issue: http://code.google.com/p/cdpedia/issues/detail?id=127_

**Comment by NicolasSandoval** [comment]
Me parece que la razon de que se descarguen despues es para descargar solo lo necesario segun la cantidad y tipos de articulos seleccionados en un formato de cdpedia.

**Comment by facundobatista** [comment]
Sí, pero tené en cuenta que en una "generación completa de CDPedias", siempre hay una que "lleva todo", así que se tienen que terminar bajando todas.

Entonces, es mejor bajar todas las imágenes al principio, y que al generar CD ya estén, al generar DVD ya estén, etc, en vez de que cada pasada trate de bajar todas de nuevo.

Lo que no me queda claro es: si luego de bajar todos los artículos, buscamos todas las imágenes y las bajamos, ¿vamos a estar bajando cosas en falso? Creo que no, pero quizás podríamos revisar eso.

Por las dudas: en este paso inicial NO se les cambia el tamaño a las imágenes, ni se "ajustan" los HTMLs... se hace sólo lo necesario como para que la generación posterior de las distintas cdpedias (CD, tarball, etc) puedan hacerse sin una conexión a internet...
